rocky/reports/report_types/aggregate_organisation_report/report.py

```diff
@@ -99,6 +99,7 @@ def post_process_data(self, data):
                         total_findings += vulnerabilities_data["summary"]["total_findings"]
                         terms.extend(vulnerabilities_data["summary"]["terms"])
                         recommendations.extend(vulnerabilities_data["summary"]["recommendations"])
+                        vulnerabilities_data["title"] = "'" + ip.split("|")[2] + "'"
```

Comment:
Why put single quotes around the IP address? It looks a bit strange to me.

Reply:
Perhaps no quotes ánd **bold** would be even better? Just throwing out an idea

Reply:
the quotes would be to differentiate from the list numbering I'd say. However, maybe the list numbering should be a different size / style / font from the contents for all these items @HeleenSG ?

Reply:
I think we can remove the single quotes. And slightly increase the space between the list numbering and the content. I believe that will be enough. 

@Donnype as the title is already bold, we can't use that to differentiate.

Reply:
New design after talking to @underdarknl  and realising this question was mainly about the ip-addresses within the table of content.

Suggested solution is to use italic text for object names (incl ip-addresses) within the toc. E.g see 6.1, 6.2 in the following design:

![Screenshot 2023-12-28 at 09 32 32](https://github.com/minvws/nl-kat-coordination/assets/11405133/d206ecc9-38dc-4cd8-8ee1-7c121d600f52)